docs/development/metagenomics_paired_end.md

```diff
@@ -0,0 +1,18 @@
+## investigation notes for updating code to allow for PE inputs into metagenomics profiling (eg for kraken, malt)
+
+see
+https://github.com/nf-core/eager/issues/945
+
+current issue is that the reads that go into mapping are not by default extracted as singletons and non-singletons, so we lose that information
+Then downstream the inputs into the krakenuniq module (even if split correctly with meta vars) don't have the correct headers to parse the PE nature of the reads (since they have all been concatenated anyways, and just were ORIGINALLY PE)
+
+So: needs to be fixed up higher (eg in bamfiltering.nf, likely with a new adjustment to the SAMTOOLS_FASTQ_UNMAPPED, SAMTOOLS_FASTQ_MAPPED, and SAMTOOLS_VIEW_BAM_FILTERING modules )
+
+ISSUE FOUND: while the outputting of PE reads is OK in bamfiltering.nf (fastq_mapped & fastq_unmapped) when overlap merging is not done cat_fastq weirdly merges singletons to one PE file and other to the other PE file, so then everything gets fucked up
```

Comment:
That sounds like a hackathon-thing to do

Reply:
let me double check this behavior and add more context of what we might need to do in a hackathon... i this was more for my own notes reference but was not edited prior to opening the PR.